files/etc/uci-defaults/99-nethsec-mwan

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+# If `mwan3.ns_default` is defined, add `last_resort=default` to it
+if uci -q show mwan3.ns_default > /dev/null; then
```

Comment:
This will override custom values.
Let's set it, only if not already set
Something like:
```
if [ -z "$(uci -q get mwan3.ns_default.last_resort)" ]; then
    uci set mwan3.ns_default.last_resort=default
fi
```

Also I think the double quotes after the `set` keyword is not required.